my-project/src/Controller/RegisterController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * VI-55 Registration
+ *
+ * @category   RegistrationController
+ * @package    Virtua_Registration
```

Comment:
@mSobonVirtua lets change all packages to Virtua_Registration, also add headers where missing